audit/projects/k8s-artifacts-prod/services/logging/logs.json

```diff
@@ -2,7 +2,6 @@
   "projects/k8s-artifacts-prod/logs/cip-audit-log",
   "projects/k8s-artifacts-prod/logs/cloudaudit.googleapis.com%2Factivity",
   "projects/k8s-artifacts-prod/logs/cloudaudit.googleapis.com%2Fdata_access",
-  "projects/k8s-artifacts-prod/logs/cloudaudit.googleapis.com%2Fsystem_event",
```

Comment:
I'm still not sure why this log is disappearing, is there a recency filter on `gcloud logging logs` I'm missing?  This should be long lived and always around...